src/components/alert-dialog/__stories__/Alert.stories.tsx

```diff
@@ -1,27 +1,22 @@
-import React from 'react';
-
 import { Meta, Story } from '@storybook/react/types-6-0';
 import { AlertContextProvider } from 'src/components/alert-dialog/AlertContext';
-import { withDesign } from 'storybook-addon-designs';
 
 import { AlertConsumer } from './AlertConsumer';
 
```

Comment:
Since you already have a good solution for grouping stories in same folder, what do you think about making this to be more consistent by moving them all out of the `__stories__` folder. And if inside of that `__stories__` folder has a component that is just being used internally for the stories, then adding a `_` prefix? 🤔